pkg/target/regolib/src.rego

```diff
@@ -341,7 +370,13 @@ does_not_match_excludednamespaces(match) {
   not always_match_ns_selectors(match)
   get_ns_name[ns]
   nss := {n | n = match.excludedNamespaces[_]}
-  count({ns} - nss) != 0
+  # count({ns} - nss) != 0
+  # JULIAN - this makes way more sense to me
+  not nss[ns]
```

Comment:
Be careful with this, if you have booleans in the set, you are not necessarily checking for membership:

https://play.openpolicyagent.org/p/ZCf7rAb5DK

Should be fine for strings though, the empty string is not "falsey"

Reply:
I see what you mean about having a boolean in the set.  Even if `false` is present in the set (and thus `mySet[false]` is defined), a rule containing that line will evaluate to `false`.  This is a painful edge case.

Like you said, I believe we are protected from this use case.  JSONSchema makes `namespaces` and `excludedNamespaces` arrays of strings, so even if there was a namespace called `"false"`, it would be safely interpreted.

Think it's ok to make the change I'm suggesting?  I find it much more semantically clear.

Reply:
I could also implement a rule that verifies that all the values in the set are strings.  Is that overkill, given my previous comment?

Reply:
Overkill :)

I think we are okay to rely on K8s schema validation

Reply:
also, the change is OK, just wanted to be sure you were aware of that edge case